ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -194,6 +194,22 @@ tests:
   optional: true
   steps:
     workflow: openshift-ci-security
+- as: e2e-agent-compact-ipv4-add-nodes
+  cluster: build05
+  optional: true
+  run_if_changed: ^(cmd|pkg|data).*/agent/
```

Comment:
Should this run if `https://github.com/openshift/oc/tree/master/pkg/cli/admin/nodeimage` path (and as an optional also go.mod is changed maybe) is changed?.